src/app/ops/show.instances.ts

```diff
@@ -40,8 +40,9 @@ export const showStudyProgramAtomic: Show<StudyProgramAtomic> = (
   }
 }
 
-export const showLabel: Show<Label> = label =>
-  label.deLabel
+export const showLabel: Show<Label> = label => {
```

Comment:
hast du aus debug gründen gemacht. kannst du wieder in eine single expression überführen